lib/capistrano/asg/tasks/rolling.rake

```diff
@@ -196,6 +197,16 @@ namespace :rolling do
         logger.info "Instance refresh(es) not completed, waiting #{wait_for} seconds..."
         sleep wait_for
       end
+      failed = false
+      completed_groups.each do |group|
+        if group.latest_instance_refresh.status == 'Failed'
+          failed = true
+          logger.error "~~ Auto Scaling Group: **#{name}** refresh failed ~~"
+        else
+          logger.info "Auto Scaling Group: **#{name}**, completed with status '#{refresh.status}'."
```

Comment:
Did you tested these changes? Because I got an error because `name` and `refresh` don't exist.

The extra loggers added seems to be redundant, because it already logged "completed with status 'Successful'". So maybe we only want to raise here when it detected status 'Failed'?

Reply:
We have but it looks like I hadn't sync'd the latest commit from our repo, apologies on that!

I've now removed that code in line with your suggestion anyway, and have confirmed latest is in this pull request :+1: